WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.h

```diff
@@ -38,5 +39,9 @@ typedef NS_ENUM(NSUInteger, ReaderCommentsSource) {
 - (void)refreshAfterCommentModeration;
 - (NSAttributedString *)cacheContentForComment:(Comment *)comment;
 - (void)trackReplyTo:(BOOL)replyTarget;
+- (void)configureCell:(CommentContentTableViewCell *)cell comment:(Comment *)comment indexPath:(NSIndexPath *)indexPath;
```

Comment:
This is mostly temporary code. I'm planning to remove it when I finish moving the rest of the code to the new controller and cells.